ote/src/cljs/ote/views/main.cljs

```diff
@@ -68,22 +68,25 @@
          flag]))]))
 
 (defn user-menu [e! name username]
-   [ui/drop-down-menu
-    {:label-style {:color "#FFFFFF"}
+  (when (not (nil? username))
```

Comment:
You can write this simpler, just `(when username ...)`

Reply:
Changed